CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # PopupBridge Android Release Notes
 
+## unreleased
+
+* Upgrade browser-switch to 2.0.0
+
+**Note:** Includes all changes in [2.0.0-beta1](#200-beta1), [2.0.0-beta2](#200-beta2), and [2.0.0-beta3](#200-beta3)
```

Comment:
```suggestion
**Note:** Includes all changes in [4.0.0-beta1](#400-beta1), [4.0.0-beta2](#400-beta2), and [4.0.0-beta3](#400-beta3)
```

Should these be 4.0.0 instead of 2.0.0?

Reply:
Yes good catch!